src/SFA.DAS.EmployerCommitmentsV2.Web/Models/Apprentice/EmployerLedChangeOfProviderRequest.cs

```diff
@@ -0,0 +1,26 @@
+﻿
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.Authorization.ModelBinding;
+using System;
+
+namespace SFA.DAS.EmployerCommitmentsV2.Web.Models.Apprentice
+{
+    public class EmployerLedChangeOfProviderRequest : IAuthorizationContextModel
```

Comment:
Not sure about the name of the class. But happy to discuss.

Reply:
CON-2518, CON-2519, CON-2520 - we have decided together to use this Model, can change it if required

Reply:
that is fine, if it is done for reason.